src/frontend/Input_warnings.ml

```diff
@@ -5,7 +5,7 @@ let init () = warnings := []
 let collect () = List.rev !warnings
 let add_warning span message = warnings := (span, message) :: !warnings
 
-let empty file =
-  add_warning Middle.Location_span.empty
-    ("Empty file '" ^ file
-   ^ "' detected; this is a valid stan model but likely unintended!")
+let empty () =
+  add_warning
+    (Preprocessor.current_location ())
+    "Empty model detected; this is a valid Stan model but likely unintended!"
```

Comment:
Here `current_location ()` is just end-of-file, right? I think this warning should either have no location (i.e. `Location_span.empty`) or a span that covers the entire file.

Reply:
Fair point. I made it the entire file